habitat/core/simulator.py

```diff
@@ -366,3 +453,11 @@ def render(self, mode: str = "rgb") -> Any:
 
     def close(self) -> None:
         raise NotImplementedError
+
+    @property
+    def index_stop_action(self):
```

Comment:
Does this properties are forcing to have stop and forward actions?

Reply:
Yes, it does force us to have stop and forward (although those are currently also forced as they are always in `SimulatorActions`)